### PR TITLE
feat: add management to initialize

### PR DESCRIPTION
**Pull request by steffenix**

### CODE REVIEW

File: scripts/deploy.py [hunk 1]
```diff
@@ -119,6 +120,7 @@ def main():
             token,
             gov,
             rewards,
+            gov,
```
Comment (fubuloubu) [inline scripts/deploy.py:123]: Do we want the script to include this as an option?

**Review by fubuloubu — APPROVED** [review]

File: scripts/deploy.py (hunk 1 above)
Comment (steffenix) [reply scripts/deploy.py:123]: Probably.

Comment (steffenix) [reply scripts/deploy.py]: updated on https://github.com/yearn/yearn-vaults/pull/271/commits/7e2de163b3bd9c8e49b6db8ee8ce733fadf2aed2

File: scripts/deploy.py [hunk 2]
```diff
@@ -92,6 +92,7 @@ def main():
 
     rewards = get_address("Rewards contract", default="treasury.ychad.eth")
     guardian = get_address("Vault Guardian", default="dev.ychad.eth")
+    management = get_address("Vault Management", default="ychad.eth")
```
Comment (poolpitako) [inline scripts/deploy.py:95]: @fubuloubu @dougstorm we need an ens for sms

Comment (fubuloubu) [reply scripts/deploy.py:95]: `brain.ychad.eth`!

**Pull request merged**